packages/stog/stog.0.15.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "zoggy@bat8.org"
+
+authors: ["Maxence Guesdon"]
+homepage: "http://zoggy.github.io/stog/"
+license: "GNU Public License version 3"
+doc: ["http://zoggy.github.io/stog/doc.html"]
+dev-repo: "https://github.com/zoggy/stog.git"
+bug-reports: "https://github.com/zoggy/stog/issues"
+tags: ["publication" "xml" "documentation" "blog" "web" "website"]
+build: [
+  ["./configure" "--prefix" prefix]
+  [make "all"]
+]
+install: [
+  [make "install-lib" "install-share"]
+]
+remove: [["ocamlfind" "remove" "stog"]]
+depends: [
+  "ocamlfind"
+  "xmlm" {>= "1.1"}
+  "xtmpl" {>= "0.12"}
+  "ocf" {>= "0.3"}
+  "ocamlnet" {>= "3.6"}
+  "higlo" { >= "0.4" }
+  "ppx_blob" {>= "0.1"}
+]
+depopts: [
+  "js_of_ocaml" { >= "2.5"}
+  "xmldiff" { >= "0.5.0" }
+  "lwt" { >= "2.4.5" }
+  "websocket" { >= "2.1" }
+  "ojs-base" { >= "0.3" }
+  "sha" { >= "1.9" }
+]
```

Comment:
Not that I know of. You could search the opam issue tracker and then raise an issue if nothing relevant comes up. As is, I don't believe this will work.